dist/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>To do list</title>
+    <script defer src="/index.bundle.js"></script>
+    <script defer src="/print.bundle.js"></script>
+    <script defer src="/status.bundle.js"></script>
+  </head>
+  <body>
+    <header>
+      <h2>Today's To-Do</h2>
+    </header>
+    <main>
+      <ul id="todo-list"></ul>
+      <input type="text" id="input-task" placeholder="Add task..." /><i
+        class="fa-sharp fa-solid fa-arrows-rotate"
+      ></i>
+
+      <br /><br />
+      <input type="checkbox" id="myCheck" value="todo" />
+      <label for="todo-list">
+        wash the dishes
+        <img src="/src/icon/verticle-menu.png" alt="menu" />
+      </label>
+      <br />
+      <input type="checkbox" id="myCheck" value="todo" />
+      <label for="todo-list">
+        complete To Do list project
+        <img src="/src/icon/return-icon.png" alt="return" /> </label
+      ><br /><br />
+      <button type="button" class="clear-btn" id="clear-btn">
+        Clear all completed
```

Comment:
- [ ] Kindly note that in this milestone you are supposed to set an array of some simple-to-do tasks (array of objects). Each task object should contain three key 
  - description [string].
  - completed [bool].
  - index: [number].
- [ ] And then Write a function to iterate over the tasks array and populate an HTML list item element for each task. Currently, your array is empty